project.yaml

```diff
@@ -1311,6 +1311,43 @@ Layer:
     properties:
       minzoom: 13
     advanced: {}
+  - name: "admin-very-low-zoom"
+    id: "admin-very-low-zoom"
+    class: ""
+    geometry: "linestring"
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+            ST_Simplify(way, !scale_denominator!/4500) as way,
```

Comment:
TBH I don't know, but if using pixel dimensions is the more common approach I'm happy to change it.